[lite][arm]fix model_optimize bug, update concat and split op, speed up

chenjiaoAngel opened the pull request.

chenjiaoAngel committed:
Merge pull request #8 from PaddlePaddle/develop

update

chenjiaoAngel committed:
add cv image process

chenjiaoAngel committed:
fix arm liunx build error

chenjiaoAngel committed:
add LITE_WITH_CV defien to make cv, test=develop

chenjiaoAngel committed:
fix cv format, annd add describe in utils/cv

chenjiaoAngel committed:
delete some Meaningless comments, test=develop

chenjiaoAngel committed:
set LITE_WITH_CV=OFF in build.sh, test=develop

chenjiaoAngel committed:
delete cv_enum.h in utils/cv, push the contents in cv_ennum.h to paddle_image_preprocess.h, test=develop

chenjiaoAngel committed:
according to reviews to redefine paddle_image_preprocess.h, test=develop

chenjiaoAngel committed:
add detailed note of flipParam, test=develop

chenjiaoAngel committed:
fix format in paddle_image_preprocess.h, test=develop

chenjiaoAngel committed:
fix error when build x86. test=develop

lite_with_X86 does not contain lite_with_cv

chenjiaoAngel committed:
fix cmake error in llite/CMakeLists.txt, missing mkdir cxx, test=develop

chenjiaoAngel committed:
Merge pull request #15 from PaddlePaddle/develop

update code

chenjiaoAngel committed:
according to review change, test=develop

chenjiaoAngel committed:
chang grb to rgb, test=develop

chenjiaoAngel committed:
Merge pull request #31 from PaddlePaddle/develop

update code

chenjiaoAngel committed:
add elemetnwise mul constant elimination and deconv+relu, deconv+batchnorm fusion, test=develop

chenjiaoAngel committed:
Merge branch 'develop' into image

chenjiaoAngel committed:
fix format, test=develop

chenjiaoAngel committed:
fix model_optimize bug, update concat and split op, speed up, test=develop

chenjiaoAngel committed:
update split speed, test=develop

chenjiaoAngel committed:
fic confilct
Merge branch 'image' of https://github.com/chenjiaoAngel/Paddle-Lite into image

chenjiaoAngel committed:
merge branch
Merge branch 'image_dev' into image

chenjiaoAngel committed:
fix format, test=develop

chenjiaoAngel committed:
Merge branch 'develop' into image

chenjiaoAngel committed:
Merge branch 'develop' into image

chenjiaoAngel committed:
add classify demo inn demo/cxx/ , test=develop

chenjiaoAngel committed:
fix formart inn mobile_classify, test=develop

chenjiaoAngel committed:
git pull code and update
Merge branch 'image' of https://github.com/chenjiaoAngel/Paddle-Lite into image

chenjiaoAngel committed:
delete some note and extra code, test=develop

chenjiaoAngel committed:
Merge branch 'develop' into image

Superjomn commented on lite/demo/cxx/mobile_classify/labels.txt:
```diff
@@ -0,0 +1,1000 @@
+n01440764 tench, Tinca tinca
+n01443537 goldfish, Carassius auratus
+n01484850 great white shark, white shark, man-eater, man-eating shark, Carcharodon carcharias
```
删掉无用数据

增大了 codebase

chenjiaoAngel replied:
已经delete了

Superjomn reviewed (changes requested):
图片和无关的数据都得清理掉，只留 code 
git的工作机制会把所有文件复制一份存在history里，会越来越大，git clone 会极慢

chenjiaoAngel committed:
remove test.jpg and labels.txt, test=develop

chenjiaoAngel committed:
pull code and update, test=develop
Merge branch 'image' of https://github.com/chenjiaoAngel/Paddle-Lite into image

Superjomn reviewed (approved):
LGTM
